src/Visible.js

```diff
@@ -244,13 +244,17 @@ class Visible extends Component {
 		/**
 		 * This event is fired when the event is compared with the last verified one and there is an element of which the visibility property has changed.
 		 * @ko 마지막으로 확인한 결과와 비교해 visibility 속성이 변경된 엘리먼트가 있을 때 발생하는 이벤트
-		 * @event eg.Visible#change
-		 * @param {Array} visible Visible elements  (the element type is `HTMLElement`) <ko>보이게 된 엘리먼트들</ko>
-		 * @param {Array} invisible Invisible elements  (the element type is `HTMLElement`) <ko>안 보이게 된 엘리먼트들</ko>
+		 * @name eg.Visible#change
+		 * @event
+		 * @type {obejct} The object of data to be sent when the event is fired<ko>이벤트가 발생할 때 전달되는 데이터 객체</ko>
+		 * @property {Array} visible Visible elements  (the element type is `HTMLElement`) <ko>보이게 된 엘리먼트들</ko>
+		 * @property {Array} invisible Invisible elements  (the element type is `HTMLElement`) <ko>안 보이게 된 엘리먼트들</ko>
+		 * @property {Boolean} isTrusted Returns true if an event was generated by the user action, or false if it was caused by a script or API call <ko>사용자의 액션에 의해 이벤트가 발생하였으면 true, 스크립트나 API호출에 의해 발생하였을 경우에는 false를 반환한다.</ko>
 		 */
 		this.trigger("change", {
 			visible: visibles,
 			invisible: invisibles,
+			isTrusted: true, // eg.Visible event is called by 'check' method.
```

Comment:
@sculove Is `isTrusted` needed although `isTrusted` is always `true`?

Reply:
Yes. `isTrusted` is coding convention of egjs.

Reply:
@sculove Thank you. 

By the way, comment `eg.Visible event is called by 'check' method.`. need to be fixed like follows `This event is called by 'check' method.`

Reply:
@jongmoon ok. thanks